rskj-core/src/test/java/co/rsk/peg/bitcoin/ErpRedeemScriptTestUtils.java

```diff
@@ -0,0 +1,167 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+import com.google.common.collect.Lists;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ErpRedeemScriptTestUtils {
+
+    public static final int PUB_KEY_LENGTH = 34; // pubKeyLength(1B) + pubKey(33B)
+
+    public static int calculateMultiSigLength(List<BtcECKey> keys) {
+        int threshold = 1;
+        int pubKeysNumber = keys.size();
+        int opN = 1;
+        int opCheckMultisig = 1;
+        return threshold + pubKeysNumber * PUB_KEY_LENGTH + opN + opCheckMultisig;
+    }
+
+    public static int calculateCSVValueLength(long csvValue) {
+        int opElse = 1;
+        int csvLengthByte = 1;
+        int csvLength = BigInteger.valueOf(csvValue).toByteArray().length;
+        int opCheckSequenceVerify = 1;
+        int opDrop = 1;
+        return opElse + csvLengthByte + csvLength + opCheckSequenceVerify + opDrop;
+    }
+
+    public static int calculateCustomRedeemScriptLength(List<BtcECKey> keys) {
+        int opNotIf = 1;
+        int opCheckSig = 1;
+        int pubKeyAndCheckSigLength = PUB_KEY_LENGTH + opCheckSig;
+        int customRedeemScriptLength = opNotIf + pubKeyAndCheckSigLength;
+
+        int opSwap = 1;
+        int opAdd = 1;
+        customRedeemScriptLength += (opSwap + pubKeyAndCheckSigLength + opAdd) * (keys.size() - 1);
+
+        int threshold = 1;
+        int opNumEqual = 1;
+        customRedeemScriptLength += (threshold + opNumEqual);
+
+        return customRedeemScriptLength;
+    }
+
+    public static void assertCustomERPRedeemScript(byte[] p2shp2wshErpCustomRedeemScriptProgram, List<BtcECKey> defaultKeys) {
```

Comment:
The same. Let's be consistent. Classes with `ERP` are named using camelCase, `Erp`. Check `ErpRedeemScript`, `P2shErpRedeemScript`, etc.